src/styles/_catalog.scss

```
@@ -1,22 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-auto-flow: row;
+  justify-content: center;
+  gap: 48px 48px;
```

you have the same gap on both axes
```suggestion
  gap: 48px;
```